fastlane/lanes/codesign.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+# Lanes related to Code Signing and Provisioning Profiles
+#
+platform :ios do
+  # Registers a new device in the App Store Portal and update all the Provisioning Profiles
```

Comment:
I _think_ we're referring to the Developer Portal here.

```suggestion
  # Registers a new device in the Developer Portal and update all the Provisioning Profiles
```